frontend/src/components/EditQuestion.js

```diff
@@ -66,6 +66,8 @@ const EditQuestion = () => {
     } catch (error) {
       if (error.response.status === 400) {
         showValidationErrorToast(error);
+      } else if (error.response.status === 409) {
+        showDuplicateQuestionErrorToast(error);
       } else {
         showServerErrorToast(error);
```

Comment:
Would a switch case be better/cleaner than multiple if-else?
```
switch (error.response.status) {
  case 400: showValidationErrorToast(error);        break;
  case 409: showDuplicateQuestionErrorToast(error); break;
  default:  showServerErrorToast(error);
}
```

Reply:
Updated code with switch-case